### ISSUE TITLE
Fix warning of missing description on `DialogContent` usage

### ISSUE THREAD

**Issue by aurelticot** [verida/vault-web#103]
`Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. `
